.circleci/config.yml

```diff
@@ -83,3 +122,6 @@ workflows:
       - release-java:
           context: release
           <<: *only-on-release
+      - release-integration-python:
```

Comment:
Why not combine the python release of `release-client-python` and `release-integration-python` into a single job named `release-python`?

Reply:
Done